### PR TITLE
feat(provider/azure): Enable dns label

### PR DESCRIPTION
**Pull request by neil-yechenwei**
Background:
Currently dns label is specified with default value while creating and updating dns label. So we add an entry to specify the custom dns label while creating and updating dns label. There is a related PR in clouddriver (https://github.com/spinnaker/clouddriver/pull/3722).

### CODE REVIEW

**Comment by neil-yechenwei** [comment]
@sbwsg , a kindly ping, please help to review it. Thanks.

File: app/scripts/modules/azure/loadBalancer/loadBalancer.transformer.js [hunk 1]
```diff
@@ -62,6 +62,9 @@ module.exports = angular
           }
 
           toEdit.probes = elb.probes;
+          if (elb.dnsName && elb.dnsName != 'dns-not-found') {
```
Comment (louisjimenez) [inline app/scripts/modules/azure/loadBalancer/loadBalancer.transformer.js]: can this use strict inequality instead? `!==`

Comment (neil-yechenwei) [reply app/scripts/modules/azure/loadBalancer/loadBalancer.transformer.js]: Sure. Updated code.

Comment (neil-yechenwei) [reply app/scripts/modules/azure/loadBalancer/loadBalancer.transformer.js]: Sure. Updated code.

**Pull request merged**